cisco_aci/assets/configuration/spec.yaml

```diff
@@ -99,6 +99,13 @@ files:
       value:
         type: string
         example: default
+    - name: enable_ndm
```

Comment:
after reviewing thibaud's [PR](https://github.com/DataDog/datadog-agent/pull/27804), maybe better to be named `send_ndm_metadata` to be more explicit

unsure of how to proceed with how metrics are perceived given the integration existed before NDM was involved, but i suppose it doesn't hurt either way to enable whichever metrics the customer would want to enable to send as well 🤔